unlock-app/src/components/interface/checkout/alpha/Lock.tsx

```diff
@@ -68,16 +69,22 @@ export function Lock({
       <div className="flex w-full mb-2 items-start justify-between">
         <h3 className="font-bold text-xl"> {name}</h3>
         {!isLoading ? (
-          <div className="grid">
+          <div className="grid text-right">
             {formattedData.cardEnabled ? (
               <>
-                <p>${(lock.fiatPricing?.usd?.keyPrice / 100).toFixed(2)}</p>
-                <p>{formattedData?.formattedKeyPrice} </p>
+                <span className="font-semibold">
+                  ${(lock.fiatPricing?.usd?.keyPrice / 100).toFixed(2)}
```

Comment:
this can gives `NaN` as result, we can move this operation to a helper to prevent this issue, and also avoid this check several times?